### PR TITLE
Update how to test ballerina content

### PR DESCRIPTION
**Pull request by yasassri**
## Purpose
Update how to test ballerina content

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ Testerina design and usage is aligned with project and package semantics of Ball
 
 ## Writing and Running Tests 
 
-You need to import the `test` package in all tests. 
+To write tests, you need to import the `test` package in all ballerina test source files.
```
[the remark this answers is not in the document]
Comment (yasassri) [reply website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: This is fixed.

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ Testerina design and usage is aligned with project and package semantics of Ball
...
 
-You need to import the `test` package in all tests. 
+To write tests, you need to import the `test` package in all Ballerina test source files. 
 
 ```
 import ballerina/test;
 ```
 
-For structured projects, it is recommended to use a structured test model that is aligned with standard package semantics. 
+For structured projects, it is recommended to use a structured test model that is aligned with standard package semantics. Structured test model consists of a seperate tests directory in a Ballerina package which allows you to isolate the source from the tests.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: Add a comma before 'which'. Normally you would have one when 'which' is in the middle of a sentence. Apply elsewhere if applicable.

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ Integration tests that are used to test the sources of combination of packages o
       packages.can.include.dots.in.dir.name.bal
 ```
 
-The test source files could have any file names. The test functions must be specified with the `@test:Config` annotation. There are no restrictions on the test function name. 
+The test source files could have any file names. The test functions are just Ballerina functions which uses a special annotation to mark the function as a test, test functions must be specified with the `@test:Config` annotation and there are no restrictions on the test function name.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: Change to: 'The test functions are just Ballerina functions that use a special annotation to mark the function as a test. Test functions must be specified with the `@test:Config` annotation and there are no restrictions on the test function name.'

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 4]
```diff
@@ -179,7 +179,7 @@ function testFunction3() {
 ```
 
 #### @test:Config {}
-The function specified following the annotation is a test function.
+The function specified following the annotation is a test function. This annotation supports following parameters.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: Change to: 'The function specified following the annotation is a test function. This annotation supports the following parameters.'

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 5]
```diff
@@ -467,13 +467,13 @@ Testerina provides the functionality to start/stop service skeletons generated f
 
 #### test:startServiceSkeleton(string packageName, string swaggerFilePath) (boolean isSuccessful)
 
-Start a service skeleton from a given Swagger definition in the given ballerina package. If it is successful returns true else returns false or throws an exception. For example: 
+Start a service skeleton from a given Swagger definition in the given Ballerina package. If it is successful returns true else returns false or throws an exception. For example:
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: Change to: 'If it is successful it returns `true`. Alternatively, it returns `false` or throws an exception.'

File: website/two-column-pages/docs/learn/how-to-test-ballerina-code.md [hunk 6]
```diff
@@ -467,13 +467,13 @@ Testerina provides the functionality to start/stop service skeletons generated f
...
 
-Start a service skeleton from a given Swagger definition in the given ballerina package. If it is successful returns true else returns false or throws an exception. For example: 
+Start a service skeleton from a given Swagger definition in the given Ballerina package. If it is successful returns true else returns false or throws an exception. For example: 
 
 ```ballerina
 boolean isSuccessful =  test:startServiceSkeleton("petstore.service.skeleton", "/tmp/petstore.yaml");
 ```
 
-When the tests are executing service skeleton related ballerina definition will be generated and started. Host names, ports you have defined in the Swagger definition will be used when starting the service. You can then invoke this service skeleton using a HTTP client endpoint, just like a normal Ballerina service.
+When the tests are executing service skeleton related Ballerina definition will be generated and started. Host names, ports you have defined in the Swagger definition will be used when starting the service. You can then invoke this service skeleton using a HTTP client endpoint, just like a normal Ballerina service.
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/how-to-test-ballerina-code.md]: The first sentence here doesn't make sense to me. Maybe a comma is missing but not sure where it should go.

Change to: 'The host names and ports you have defined in the Swagger definition will be used when starting the service.'

**Pull request merged**